source/features/latest-tag-button.tsx

```diff
@@ -37,26 +39,59 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 	return tags[0];
 }, {
 	expiration: 1,
-	cacheKey: () => __featureName__ + ':' + getRepoURL()
+	cacheKey: () => __featureName__ + '_tags:' + getRepoURL()
 });
 
-function getTagLink(latestRelease: string): HTMLAnchorElement {
+async function currentBranchIsBleedingEdge(latestTag: string, defaultBranch: string, currentBranch: string): Promise<boolean> {
+	// Only default branch can be cutting edge (i.e., ahead of latest release)
+	if (currentBranch !== defaultBranch) {
+		return false;
+	}
+
+	// Find the last commits from the default branch and latest tag
+	const repository = await api.v4(`
+		repository(${getRepoGQL()}) {
+			latestTag:object(expression: "${latestTag}") {
+				... on Commit {
+					oid
+				}
+			  },
+			  defaultBranch:object(expression: "${defaultBranch}") {
```

Comment:
Is there any other way to just get the latest commit or `HEAD` of a repo? I think this works `defaultBranchRef.target.oid`

Reply:
Not sure what you mean by this one - is there already a util/function that knows about the defaultBranch oid?
The only thing I could find in the codebase was the getDefaultBranch(), but that retrieves the  name only

Reply:
It’s the API itself: use

```GQL
repository(…) {
  defaultBranchRef {
    target {
      oid 
    }
  }
}
```

Reply:
Ah, I understand. Cheers